packages/terra-tabs/src/StructuralTabs.scss

```diff
@@ -0,0 +1,129 @@
+// stylelint-disable selector-max-compound-selectors
+// stylelint-disable max-nesting-depth
+:local {
+  .structural {
+    background-color: var(--terra-tabs-structural-container-background-color, #fff);
+
+    > :nth-child(2) {
```

Comment:
What's this selector used for. Might be good to add a comment to help with clarity.